plugins/org.eclipse.osee.orcs.core/src/org/eclipse/osee/orcs/core/internal/transaction/TransactionFactoryImpl.java

```diff
@@ -370,75 +408,75 @@ public int compare(ChangeReportRowDto o1, ChangeReportRowDto o2) {
    @Override
    public boolean undoTx(BranchId branch, TransactionId transaction) {
       //@formatter:off
-String query =
```

Comment:
Can this formatting get removed? So we don't conflict with the mim gamma ids stuff(soon^tm)

Reply:
No, what I'm asking is, please don't format this file, as you are going to cause merge conflicts on the mim gamma id branch where we fixed a bunch of issues related to transactions.

Reply:
oh gotcha... ok updated it.